client-it/src/test/java/io/streamnative/oxia/client/it/OxiaClientIT.java

```diff
@@ -229,6 +230,29 @@ void test() throws Exception {
                 .isEqualTo(24);
     }
 
+    @Test
+    public void testGetFloorCeilingWithPartitionKey() throws Exception {
+        @Cleanup
+        SyncOxiaClient client = OxiaClientBuilder.create(oxia.getServiceAddress()).syncClient();
+
+
+        GetResult gr = client.get("a", Set.of(GetOption.PartitionKey("a"), GetOption.ComparisonCeiling));
```

Comment:
These keys were added by previous tests and might or might not have the partition-key set. Therefore, the routing to a shard could be different. 

Take a look at https://github.com/streamnative/oxia-java/blob/main/client/src/main/java/io/streamnative/oxia/client/AsyncOxiaClientImpl.java#L480-L486 to see how passing a partition-key changes the Get operation.